.github/workflows/nr1_lib_deprecations.yml

```
@@ -0,0 +1,26 @@
+name: NR1 Library Deprecation Checks
+
+on: [push, workflow_dispatch]
+
+jobs:
+  nr1_lib_deprecations:
+    name: Run NR1 Library Deprecation Checks
+    runs-on: ubuntu-latest
+    steps:
+      - name: Test Default Branch
+        id: default-branch
+        uses: actions/github-script@v2
+        with:
+          script: |
+            const data = await github.repos.get(context.repo)
+            return data.data && data.data.default_branch === context.ref.split('/').slice(-1)[0]
+      - name: Checkout Self
+        if: ${{ steps.default-branch.outputs.result == 'true' }}
+        uses: actions/checkout@v2
+      - name: Run Repolinter
+        if: ${{ steps.default-branch.outputs.result == 'true' }}
+        uses: newrelic/repolinter-action@v1
+        with:
+          # FIXME: Replace with the appropriate ruleset URL
+          config_url: https://raw.githubusercontent.com/newrelic/.github/main/repolinter-rulesets/nr1-lib-deprecations.yml
+          output_type: issue
```

At the moment repolinter-action doesn't work correctly if there are two running at once, since they both attempt to edit the same issue. While I can look into fixing this problem in repolinter-action such that it works w/o configuration, a quick fix would be to tell this action to use a different label name so that the issues are distinct:
```suggestion
          output_type: issue
          label_name: nr1-deprecation
```